### ISSUE TITLE
Getting error: [ErrorDetail(string='Too many uploads to this commit.', code='invalid')]

### ISSUE THREAD

**Issue by JoanFM** [codecov/codecov-action#228]
Hello team!

I am seeing in our CI this message when trying to upload the coverage files for a commit. We are parallelizing our CI quite heavily and thus we are uploading quite some reports for a single commit.

I would like to know what is the limit for which the error happens, and if there is a solution or workaround for this problem.

Thank you very much

**Comment by rollbear** [comment]
Same. Here's an example:

https://github.com/rollbear/trompeloeil/runs/1949799000?check_suite_focus=true

They do fail quite intermittently, though. It can be the first upload that fails, and later ones succeed, for the same commit.

**Comment by altendky** [comment]
Just ended up here on my latest search on this topic...  I figured I'd share my knowledge so far.  I am getting this over in https://github.com/altendky/ssst/issues/38.  Note that I do not use this action.  It showed up a few days ago in my nightly scheduled run.  I do not get it in other repositories, including QTrio which has a bigger matrix.  There are a couple reports directly to Codecov (links in the above issue).  It seems more like a bug to me than a policy change since there isn't wide-spread freak-out, as far as I can tell, nor consistency.

I did do an intra-workflow collection setup in https://github.com/riptideio/pymodbus recently and started to make an action for it...  https://github.com/altendky/coverage-artifact but I had forgotten that actions can't use other actions, so, so much for that for the moment.  The idea being to make it easy to collect coverage records from several jobs into an actions artifact as well as to allow single-point uploading for the whole workflow.  Oh well.
